pkg/test/devstack/submit_test.go

```diff
@@ -59,7 +59,7 @@ func (suite *DevstackSubmitSuite) TestEmptySpec() {
 	require.Error(suite.T(), missingSpecError)
 
 	j = &model.Job{}
-	j.Spec = model.Spec{Engine: model.EngineDocker}
+	j.Spec = model.Spec{EngineDeprecated: model.EngineDocker, EngineSpec: model.NewDockerEngineSpec("", nil, nil, "")}
```

Comment:
It would probably be better to make this a noop engine as I believe the intention is to assert that the spec is invalid due to an empty deal structure, thoughts?